doc/api/toolkits/mplot3d.rst

```diff
@@ -0,0 +1,124 @@
+.. _toolkit_mplot3d-api:
+
+***********
+mplot3d API
+***********
+
+.. contents::
+      :backlinks: none
+
+.. currentmodule:: mpl_toolkits.mplot3d
+
+.. _toolkit_mplot3d-axesapi:
+
+:mod:`~mpl_toolkits.mplot3d.axes3d`
+===================================
+
+.. note::
+   Significant effort went into bringing axes3d to feature-parity with
+   regular axes objects for version 1.1.0. However, more work remains.
+   Please report any functions that do not behave as expected as a bug.
+   In addition, help and patches would be greatly appreciated!
+
+.. autosummary::
+   :toctree: ../_as_gen
+   :template: autosummary.rst
+
+   axes3d.Axes3D
+
+
+.. _toolkit_mplot3d-axisapi:
+
+:mod:`~mpl_toolkits.mplot3d.axis3d`
+===================================
+
+.. note::
+   Historically, axis3d has suffered from having hard-coded constants
+   controlling the look and feel of the 3D plot. This precluded user
+   level adjustments such as label spacing, font colors and panel colors.
```

Comment:
The entire note is useful information to the end-user that wants to fully customize the mplot3d properties. The note has been there since v1.1.0. The kludge dictionary still exists, hence the note is still needed to document it.

I think we have spent enough time discussing this note. If you wish to eliminate this note, then please re-implement all of axis3d so that it would work like a normal axis object.